dilation-protocol.md

```diff
@@ -9,34 +9,44 @@ must select the best one to use, and cleanly shut down the others. To make
 the dilated connection *durable*, this side must also decide when the
 connection has been lost, and then coordinate the construction of a
 replacement. Within this connection, a series of queued-and-acked subchannel
-messages are used to open/use/close the application-visible subchannels.
+messages are used to open/use/close the application-visible subchannels.~~
+
+TODO rewrite that paragraph from scratch ~piegames
+
+## Overview
+
+A dilated Wormhole connection involves several moving parts.
+
+The Mailbox on the rendezvous server remains accessible to the application.
+It is furthermore used by the Dilation implementation to exchange internal
+coordination messages.
+
+The Transit connection (TODO rename) is now managed by the Dilation
+implementation. It will automatically be established in the beginning.
```

Comment:
"It will automatically be established in the beginning..." looks very confusing to me. What exactly is meant here?